Library/Homebrew/cmd/cask.rb

```diff
@@ -1,10 +1 @@
-$LOAD_PATH.unshift("#{HOMEBREW_LIBRARY_PATH}/cask/lib")
-require "hbc"
-
-module Homebrew
-  module_function
-
-  def cask
-    Hbc::CLI.run(*ARGV)
```

Comment:
I think `odie "The cask command requires macOS." if OS.linux?` above this line could be sufficient for this whole PR.

Reply:
This is literally what Linuxbrew does right now: https://github.com/Linuxbrew/brew/blob/master/Library/Homebrew/cmd/cask.rb#L8

I will submit a PR that ports Linuxbrew's changes. Also, I believe it should be `unless OS.mac?` because one can't expect cask to work anywhere else, right?